wweval/R/eval_post_process.R

```diff
@@ -44,16 +51,15 @@ eval_post_process_ww <- function(config_index,
   eval_hosp_data <- load_object("eval_hosp_data", output_file_suffix)
   input_ww_data <- load_object("input_ww_data", output_file_suffix)
   eval_ww_data <- load_object("eval_ww_data", output_file_suffix)
-  ww_fit_obj <- load_object("ww_fit_obj", output_file_suffix)
+  ww_fit_obj_wwinference <- load_object("ww_fit_obj", output_file_suffix)
+  ww_fit_obj <- ww_fit_obj_wwinference$fit$result
```

Comment:
I thought this would return what is identical to `ww_fit_obj` above but it doesn't seem to as I can't get the same elements out of it exactly (e.g. flags, diagnostics etc see below).

Reply:
Were you able to debug this?

Reply:
Yes! It was just a brain fart on my part, I had thought the original `wweval` implementation was returning the raw stan object but it actually is a result of a `sample_model` function that I had written that calculated flags and a few other outputs, and so I added the generation of those outputs to this